docs/API.md

```diff
@@ -21,7 +21,7 @@ etc., still work on the resulting hypertable.
 |---|---|
 | `partitioning_column` | Name of an additional column to partition by. If provided, `number_partitions` must be set.
 | `number_partitions` | Number of partitions to use when `partitioning_column` is set. Must be > 0.
-
+| `chunk_time_interval` | Interval in event time (micro seconds) that a chunk covers. Must be > 0. Default is 1 month.
```

Comment:
I think we should specify that the units are micro seconds for TIMESTAMP and TIMESTAMPTZ time columns. Also maybe "that each chunk covers".